nixos/modules/services/mail/pfix-srsd.nix

```diff
@@ -0,0 +1,53 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  secretsFile = pkgs.writeText "pfix-srsd-secrets" config.services.pfix-srsd.secret;
+
+in
+
+{
+
+  ###### interface
+
+  options = {
+
+    services.pfix-srsd = {
+      enable = mkOption {
+        default = false;
+        description = "Whether to run the postfix sender rewriting scheme daemon.";
+      };
+
+      domain = mkOption {
+        description = "The domain for which to enable srs";
+      };
+
+      secret = mkOption {
+        description = "The secret data used to encode the SRS address";
```

Comment:
How important is it that this remains secret? If at all, consider not copying it into the store.

Reply:
I think it is rather important -- possession of this secret means you can forge bounces through this mail server.  Looking into duplicating the approach used by `postsrsd.nix`.